alembic/env.py

```
@@ -7,20 +7,21 @@
 from sqlalchemy import engine_from_config, pool
 
-# Ensure application modules are importable when Alembic runs standalone.
-import sys
+from alembic import context
 
 ROOT_DIR = Path(__file__).resolve().parents[1]
 if str(ROOT_DIR) not in sys.path:
     sys.path.insert(0, str(ROOT_DIR))
 
-from core.db import Base, DATABASE_URL  # noqa: E402
+from core.db import DATABASE_URL, Base  # noqa: E402
```

Ruff (0.13.1)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)